packages/11ty/_includes/components/figure/image/sequence-panel.js

```diff
@@ -26,10 +25,10 @@ module.exports = function(eleventyConfig) {
       id,
       manifestId,
       margin='',
-      preset='responsive',
+      preset='static',
       startCanvas,
-      textEnabled='true',
-      textSelectionEnabled='true',
+      textEnabled='false',
+      textSelectionEnabled='false',
```

Comment:
these don't actually do anything, left over from when I was trying to make digirati `<sequence-panel>` work